File: test_files/java/SimpleAutocomplete.java
public class SimpleAutocomplete {
    static String formatName(String first, String last) {
        return first + " " + last;
    }

    static String greet(String fullName) {
        return "Hello, " + fullName + "!";
    }

    public static void main(String[] args) {
        String name = formatName("Rina", "Patel");
        System.out.println(name);

        String message = greet(
    }

    static void suffixMidlineDemo() {
        String name = formatName("Rina", "Patel");
        String suffixMessage = greet();
        System.out.println(suffixMessage);
    }

    static String maskedWordDemo() {
        String name = formatName("Rina", "Patel");
        return nam;
    }
}

File: test_files/typescript/simple_autocomplete.ts
export function titleCase(input: string): string {
  return input
    .split(" ")
    .filter(Boolean)
    .map((word) => word[0].toUpperCase() + word.slice(1).toLowerCase())
    .join(" ");
}

export function welcome(name: string): string {
  return `Welcome, ${name}!`;
}

function runDemo(): void {
  const normalized = titleCase("sAm lee");
  console.log(normalized);

  const message = welcome(
}

function suffixMidlineDemo(): void {
  const normalized = titleCase("sAm lee");
  const suffixMessage = welcome();
  console.log(suffixMessage);
}

function maskedWordDemo(): string {
  const normalized = titleCase("sAm lee");
  return normaliz;
}

runDemo();
